README.md

```diff
@@ -123,6 +129,17 @@ instance to use the screen sharing functionality you will have to
 provide HTTPS access to both the files and the Janus gateway, like shown
 in the [deployment instructions](DEPLOYMENT.md).
 
+## Troubleshooting
+
+When jangouts server do not work, there is wide variety of ways how to ways how to debug it.
```

Comment:
well, for me jangouts server is whole service including all js and web services. Attaching to janus server, etc. So thats why I use jangouts...just first trouble is janus specific, as it doesn't work.